containers/allan_dip_bot/run_bot.py

```diff
@@ -297,6 +327,12 @@ def main() -> None:
     bot_type: str = args.bot_type
     sleep_delay: bool = args.no_sleep_delay
     discount_factor: float = args.discount_factor
+    invasion_coef: float = args.invasion_coef
+    conflict_coef: float = args.conflict_coef
+    invasive_support_coef: float = args.invasive_support_coef
+    conflict_support_coef: float = args.conflict_support_coef
+    friendly_coef: float = args.friendly_coef
+    unrealized_coef: float = args.unrealized_coef
```

Comment:
These all need to be `Optional` or have default values set.

Reply:
fixed